src/orders/src/main/java/mushop/orders/services/OrdersService.java

```diff
@@ -24,6 +24,9 @@
 import static mushop.orders.controllers.OrdersController.OrderFailedException;
 import static mushop.orders.controllers.OrdersController.PaymentDeclinedException;
 
+import mushop.orders.config.MonitoringConfiguration;
+import com.oracle.bmc.monitoring.MonitoringClient;
```

Comment:
Orders Service depending on MonitoringClient seems too tightly coupled to OCI and a very specific monitoring stack. Should decouple the implementation, or more likely spin off in to a separate async service

Reply:
yes. I was thinking if a well tested image would be just good ex: iad.ocir.io/oracle/ateam/mushop-orders-monit:2.2.3